packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -180,6 +189,11 @@ class ExpansionTile extends StatefulWidget {
   /// Used to override to the [ListTileTheme.textColor].
   final Color? collapsedTextColor;
 
+  /// Defines the position of [trailing], relative to text.
```

Comment:
This is a little confusing, as it seems that we've provided this parameter so that one can make trailing mean leading (without changing the meaning of leading).  Perhaps it would be better to say: "Typically used to force the expansion arrow icon to the tile's leading or trailing edge.